CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - helm.v3.Release: Improved cancellation support (https://github.com/pulumi/pulumi-kubernetes/pull/2579)
 - Update Kubernetes client library to v0.28.2 (https://github.com/pulumi/pulumi-kubernetes/pull/2585)
 - Fix for Helm Import regression (https://github.com/pulumi/pulumi-kubernetes/pull/2605)
+- Improved search functionality for Helm Import (https://github.com/pulumi/pulumi-kubernetes/pull/2610)
```

Comment:
Can you rebase this PR on top of the latest master? This line might end up in the wrong place if we don't rebase correctly.